index.html

```diff
@@ -13,43 +13,132 @@
-        <div class="container">
-            <section>
-                    <div class="row">
-                        <div class="col-sm-6 col-md-3 mb-3 mb-sm-0">
-                            <div class="profile-card">
-                                <img src="./images/profile-image.jpg" alt="Profile Image" id="profile-pic">
-                                <input type="file" id="input-file" accept="image/*">
-                            </div>
+        <section>
+                <div class="row">
+                    <div class="col-sm-6 col-md-3 mb-3 mb-sm-0">
+                        <div class="profile-card">
+                            <img src="./images/profile-image.jpg" alt="Profile Image" id="profile-pic">
+                            <input type="file" id="input-file" accept="image/*">
                         </div>
-                        <div class="col-sm-6 col-md-9">
-                            <div class="card-body">
-                                <div class="input-group mb-1">
-                                    <input type="text" class="form-control" placeholder="Name" aria-label="Name">
-                                </div>
-                                <div class="input-group mb-1">
-                                    <input type="text" class="form-control" placeholder="Mobile Number" aria-label="Mobile Number">
-                                </div>
-                                <div class="input-group mb-1">
-                                    <input type="text" class="form-control" placeholder="Email" aria-label="Email">
-                                </div>
-                                <div class="input-group mb-1">
-                                    <input type="text" class="form-control" placeholder="Address" aria-label="Address">
-                                </div>
+                    </div>
+                    <div class="col-sm-6 col-md-9">
+                        <div class="card-body">
+                            <div class="input-group mb-1">
+                                <input type="text" class="form-control" placeholder="Name" aria-label="Name">
+                            </div>
+                            <div class="input-group mb-1">
+                                <input type="text" class="form-control" placeholder="Mobile Number" aria-label="Mobile Number">
+                            </div>
+                            <div class="input-group mb-1">
+                                <input type="text" class="form-control" placeholder="Email" aria-label="Email">
+                            </div>
+                            <div class="input-group mb-1">
+                                <input type="text" class="form-control" placeholder="Address" aria-label="Address">
                             </div>
                         </div>
                     </div>
-            </section>
-    </div>
+                </div>
+        </section>
+</div>
+
+
```

Comment:
Remove this whole div containers. No need of this in your task There is a another task for this section.

Reply:
This changes from Ajay D.